### PR TITLE
Facility to provision primordial accounts for fullnodes in genesis block

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
The testnet scripts do not support creation and assignment of primordial account for nodes.

#### Summary of Changes
Updated testnet scripts for
1. Provide a YAML file with account pubkey and corresponding lamports for external nodes
2. Configure internal nodes with accounts and lamports provisioned in genesis block 
3. Fixed genesis block code to use simple strings for YAML parsing

### CODE REVIEW

**Comment by pgarg66** [comment]
https://github.com/solana-labs/solana/issues/4539

File: net/remote/remote-node.sh [hunk 1]
```diff
@@ -77,6 +80,20 @@ local|tar)
       export SOLANA_CUDA=1
     fi
     set -x
+    rm -rf ./solana-node-keys
+    rm -rf ./solana-node-stakes
+    mkdir ./solana-node-stakes
+    if [[ -n $stakeNodesInGenesisBlock ]]; then
+      for i in $(seq 1 "$numNodes"); do
+        solana-keygen new -o ./solana-node-keys/"$i"
```
Comment (mvines) [inline net/remote/remote-node.sh]: Add an -f here I think, `solana-keygen new -f -o ./solana-node-keys/"$i"`, to handle `./net.sh restart`

File: net/remote/remote-node.sh [hunk 2]
```diff
@@ -119,6 +139,10 @@ local|tar)
       args+=(--enable-rpc-exit)
     fi
 
+    if [ -f ~/solana/fullnode-identity.json ]; then
```
Comment (mvines) [inline net/remote/remote-node.sh]: Nit: `if [[ -f ~/solana/fullnode-identity.json ]]; then`

File: net/remote/remote-node.sh (hunk 1 above)
Comment (pgarg66) [reply net/remote/remote-node.sh]: seems -f is not a valid option..

Comment (mvines) [reply net/remote/remote-node.sh]: https://github.com/solana-labs/solana/blob/master/keygen/src/keygen.rs#L35-L39

Comment (pgarg66) [reply net/remote/remote-node.sh]: Somehow it gave me an error when I tried to run in on testnet.

Comment (pgarg66) [reply net/remote/remote-node.sh]:
```solana-keygen new -f -o ./solana-node-keys/0
error: Found argument '-f' which wasn't expected, or isn't valid in this context
```

**Pull request merged**